### PR TITLE
Add documentation for the Node.js HTTP REST CRUD Booster

### PR DESCRIPTION
**Pull request by lholmquist**
Like i said last time,   i think i got everything, maybe :)

This adds in the docs for the nodejs-crud-booster here: https://github.com/bucharest-gold/nodejs-rest-http-crud

### CODE REVIEW

**Comment by rhoads-zach** [comment]
LGTM, merging.

Nice work @lholmquist !

**Comment by rhoads-zach** [comment]
merged: e8e1cc5ffa9a741bc53fb38edbce37c925eff714

File: docs/topics/crud-mission-deploy-booster.adoc [hunk 1]
```diff
@@ -44,14 +44,27 @@ my-database-1-deploy   0/1       Completed   0         53s
...
+
+ifdef::http-api-nodejs[]
+. Use `npm` to start the deployment to {OpenShiftLocal}.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ npm install && npm run openshift
+----
++
+These commands use the Nodeshift `npm` module to install your dependencies and launch the S2I process on your {OpenShiftLocal} and start the pod.
```
Comment (Ladicek) [inline docs/topics/crud-mission-deploy-booster.adoc]: I think we changed this formulation recently (?). This sounds like Nodeshift is used to install dependencies, which isn't exactly correct.

Comment (rhoads-zach) [reply docs/topics/crud-mission-deploy-booster.adoc]: @Ladicek tracked in #557

Comment (Ladicek) [reply docs/topics/crud-mission-deploy-booster.adoc]: Ah, thanks!

**Pull request closed without merging**